rfc/rfc-000-lightweightRFCprocess.md

```diff
@@ -0,0 +1,76 @@
+# RFC-000: Lightweight RFC Process for riff
+
+**To be Reviewed By:** Nov 15th 2019
+
+**Authors:** Swapnil Bawaskar
+
+**Status:** **Active** | Accepted | Dropped | Superseded
+
+**Pull Request URL:** https://github.com/projectriff/riff/pull/1358
+
+**Superseded by:** N/A
+
+**Supersedes:** N/A
+
+**Related:** N/A
+
+
+## Problem
+As the scope of the project expands and the number of moving pieces increases, we need a process to gather feedback on the changes proposed to the architecture and interaction of various components in the system.
+
+This document will refer to that process as the “Request For Comments (RFC) process”.
+
+## Solution
+The proposed solution to address the problem described above is to have an individual author or a group of authors submit a proposal to the community in order to gather feedback and achieve consensus. The RFC follows the same format as used by this proposal.
+
+Much inspiration for this proposal has been drawn from [Apache Geode's RFC process](https://cwiki.apache.org/confluence/display/GEODE/Lightweight+RFC+Process), which in turn draws from Phil Calçado’s [Structured RFC Process](https://philcalcado.com/2018/11/19/a_structured_rfc_process.html).
+
+All RFCs are submitted via PRs to the [github.com/projectriff/riff](https://github.com/projectriff/riff) repo which will be merged when:
+- The `Reviewed By` date has elapsed
+- The RFC author has summarized the discussion in the PR thread.
+- concensus has been reached about the state ("Accepted/Dropped").
+
+### Collaboration
+Comments and feedback should be provided on the PR.
+
+Authors should address all comments by the deadline. This doesn't mean every comment and suggestion must be accepted and incorporated, but they should be carefully read and responded to.
+
+Every RFC is in one of the following phases:
+* **Active**: The RFC has been drafted and is open for comments. When making the PR, the doc should start out in this state.
+* **Accepted**: All the comments have been addressed and the proposed changes have been agreed upon. The implementation may start after this point.
+* **Dropped**: The changes proposed on this RFC were not agreed upon and no implementation will follow.
+* **Superseded**: The changes proposed on this RFC aren't in effect anymore, the document may be kept for historical purposes and there is a new RFC that’s more current.
+
+### Approval
+The proposal should be posted with a date by which the author would like to have the approval decision. How much time is given to comment depends on the size and complexity of the proposed changes, but ideally should be from 2 days to a week.
```

Comment:
I suggest omitting this this decision-deadline part. It adds weight to a process, and it invites pressure on maintainers and could be covered by the same conventions (determined by maintainers) for how to manage PR reviews.

Reply:
removed.